### PR TITLE
Interval(-oo,oo) contains oo and -oo

### LINKED ISSUE #10326: Interval(-oo, oo) contains oo
Related : #9706 

```python
In [ ]: foo = Interval(-oo, oo)

In [ ]: foo.contains(-oo)
Out[ ]: True

In [ ]: foo.contains(oo)
Out[ ]: True

```

Ping @asmeurer

### PR DESCRIPTION
**Pull request by shaoron**
Fix #10326 issue
Just add 2 lines to handle (-oo,oo) case

### CODE REVIEW

File: sympy/sets/sets.py [hunk 1]
```diff
@@ -969,6 +969,8 @@ def _contains(self, other):
 
         if self.start is S.NegativeInfinity and self.end is S.Infinity:
             if not other.is_real is None:
+                if other is S.NegativeInfinity or other is S.Infinity:
+                    return false
```
Comment (kshitij10496) [inline sympy/sets/sets.py]: I think, we can check for `other` belonging to extended reals without even checking `self` for the same.

**Comment by shaoron** [comment]
Ok, I move this check.

**Comment by asmeurer** [comment]
Please add tests.

**Comment by shaoron** [comment]
I've never written tests. Something like this?

**Comment by asmeurer** [comment]
Yep, perfect.

**Pull request merged**